test/dataset/split/test_split.py

```diff
@@ -78,3 +77,19 @@ def test_splitter():
     train, validation = splitter.rolling_split(dataset.train, windows=3)
     for i in range(3):
         assert len(validation[1][i][FieldName.TARGET]) == max_history
+
+
+def test_split_mult_freq():
+    splitter = DateSplitter(
+        prediction_length=1, split_date=pd.Timestamp("2021-01-01")
+    )
+
+    splitter.split(
```

Comment:
Do you want to test whether the split result is what we expected?

Reply:
It's enough that it doesn't fail :)